db/migrate/20190417171019_add_inbox_instructions_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddInboxInstructionsToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :inbox_guidelines, :string
```

Comment:
This should probably be `:text` instead of `:string` even though there's not much difference at the PostgreSQL level, it might matter for Rails since it's used to display a paragraph of text

Reply:
I had the thought that brevity is a feature here. If people are needing longer inbox instructions, we can make a change. But 256 chars should suffice based on how this should be used IMO.

Reply:
@benhalpern got it! It make sense.

Just know that Rails isn't going to validate the length. You can actually fit as much text as you want in a string field, unless there's an explicit length validator. Rails relies on the DB for this and PostgreSQL doesn't make a difference